lib/runtime/juttle-utils.js

```diff
@@ -50,7 +50,12 @@ function parseTime(points, timeField, context) {
         }
 
         var time = pt[timeField || 'time'];
-        if (time) { pt.time = time; }
+        if (time) {
+            pt.time = time;
+            if (timeField !== 'time') {
```

Comment:
Not sure this really matters, but this would only actually remove the timeField if it is truthy. So if someone had a -timeField with values that were undefined, the points with undefined timeFields would remain, even though the time  values would be undefined.